src/rust/lib/system/web/src/lib.rs

```diff
@@ -203,7 +203,13 @@ pub fn get_webgl2_context(canvas:&HtmlCanvasElement) -> WebGl2RenderingContext {
     let options = js_sys::Object::new();
     js_sys::Reflect::set(&options, &"antialias".into(), &false.into()).unwrap();
     let context = canvas.get_context_with_context_options("webgl2",&options).unwrap().unwrap();
-    context.dyn_into().unwrap()
+    let context : WebGl2RenderingContext =  context.dyn_into().unwrap();
+    // Add this extension so we can use RGB32f textures for rendering object ids in our picking
+    // pipeline.
+    context
+        .get_extension("EXT_color_buffer_float")
```

Comment:
Could you please update comment in `texture/types/relations.rs` that this extension makes some changes to the table provided there?

Reply:
Actually we don't need that extension anymore! Is removed now.

Reply:
Still, please add the comment to `relations.rs`. It might be needed in future and we will forget about it.